### ISSUE TITLE
Mac Os High Sierra(10.13.3)

### ISSUE THREAD

**Issue by licarigianluca** [adrienverge/openfortivpn#323]
Hi,

I'm experiencing issues on my Mac running High Sierra.

`sudo openfortivpn -c Downloads/vpn/vpn.conf`
````
INFO:   Connected to gateway.
INFO:   Authenticated.
INFO:   Remote gateway has allocated a VPN.
INFO:   Got addresses: [10.xx.xx.xx], ns [xx.xx.xx.xx, xx.xx.xx.xx]
INFO:   Interface ppp0 is UP.
INFO:   Setting new routes...
```

Trying to modify routing table... and if I hit CTRL-C

`^CINFO:   Cancelling threads...
`

and ppp0 is still up

```
ppp0: flags=8051<UP,POINTOPOINT,RUNNING,MULTICAST> mtu 1354
	inet xx.xx.xx.xx --> 192.xx.xx.xx netmask 0xff000000
```

If I run with the parameter "--no-routes"


```
VPN account password:
INFO:   Connected to gateway.
INFO:   Authenticated.
INFO:   Remote gateway has allocated a VPN.
INFO:   Got addresses: [10.xx.xx.xx], ns [xx.xx.xx.xx, xx.xx.xx.xx]
INFO:   Interface ppp1 is UP.
INFO:   Adding VPN nameservers...
INFO:   Tunnel is up and running.
```

and CTRL-C works fine

```
^CINFO:   Cancelling threads...
INFO:   Setting ppp interface down.
INFO:   Removing VPN nameservers...
INFO:   pppd: The link was terminated by the modem hanging up.
INFO:   Terminated pppd.
INFO:   Closed connection to gateway.
INFO:   Logged out.
```

If you need more info please write me back.

THX

**Comment by DimitriPapadopoulos** [comment]
I understand **Ctrl+C** will not destroy `ppp0` unless option `--no-routes` is used. Is that correct?

Which version of *openfortivpn* is this? Looks like 1.7.0. There have been related issues in the past:
#132 *fix lock status when fail to spawn pppd in macOS.*
#135 *fix core cause of openfortivpn is locked when spawning `ppp` is failed.*

**Comment by DimitriPapadopoulos** [comment]
I have access to a mac to build _openfortivpn_ but I cannot test for lack of access to a FortiGate device from that mac. Perhaps running _openfortivpn_ with `-v` or `-v -v` might give some clues.

**Comment by licarigianluca** [comment]
Hi,
 
sorry for the late reply. I tried with a Mac Os High Sierra 10.13.3 then I have upgrade the os to 10.13.4 and the problem solved. I delete all routes manually too after the upgrade. I don't know if the problem was solved by the upgrade or by the deletion of the ruotes. :)
I'm very happy now to use this software on my Mac too because it semplifies the way my computer connects to the VPN gateway.

Thanks

G
